capabilityquerier/cmd/main.go

```diff
@@ -80,17 +79,16 @@ func queryEndpoints(message []byte, args *map[string]interface{}) error {
 
 	urlString := string(message)
 	// Specifically query the FHIR endpoint metadata
-	metadataURL, err := url.Parse(urlString)
+	fhirURL, err := url.Parse(urlString)
 	if err != nil {
+		log.Warnf("Error parsing URL string %s\n", urlString)
```

Comment:
I don't think this log message needs to be here because the returned error message is already printed out on the ProcessMessage error channel

Reply:
but also this is incredibly minor so you don't have to change it